tests/kustomize/test_runner.py

```diff
@@ -23,6 +23,33 @@ def test_record_relative_path_with_relative_dir(self):
         dir_rel_path = os.path.relpath(scan_dir_path).replace('\\', '/')
 
         runner = Runner()
+        runner.templateRendererCommand = "kustomize"
+        runner.templateRendererCommandOptions = "build"
+        checks_allowlist = ['CKV_K8S_37']
+        report = runner.run(root_folder=dir_rel_path, external_checks_dir=None,
+                            runner_filter=RunnerFilter(framework='kustomize', checks=checks_allowlist))
+
+        all_checks = report.failed_checks + report.passed_checks
+        self.assertGreater(len(all_checks), 0)  # ensure that the assertions below are going to do something
+        for record in all_checks:
+            # Kustomize deals with absolute paths
+            #self.assertEqual(record.repo_file_path in record.file_path)
+            self.assertIn(record.repo_file_path, record.file_path)
+
+    @unittest.skipIf(os.name == "nt", "Skipping Kustomize test for windows OS.")
+    def test_record_relative_path_with_direct_oberlay(self):
+        # test whether the record's repo_file_path is correct, relative to the CWD (with a / at the start).
+
+        # this is just constructing the scan dir as normal
+        current_dir = os.path.dirname(os.path.realpath(__file__))
+        scan_dir_path = os.path.join(current_dir, "runner", "resources", "overlays", "dev")
+
+        # this is the relative path to the directory to scan (what would actually get passed to the -d arg)
+        dir_rel_path = os.path.relpath(scan_dir_path).replace('\\', '/')
```

Comment:
any reason for `.replace(...)` at the end?

Reply:
This was a suggestion from another runners test boilerplate by @schosterbarak

Reply:
maybe it is related to some Windows shenanigans 🤷